package.json

```diff
@@ -21,6 +18,7 @@
 		"@vite-pwa/nuxt": "^0.0.8",
 		"@vueuse/core": "^10.1.2",
 		"@vueuse/nuxt": "^10.1.2",
+		"canvas-constructor": "^7.0.1",
```

Comment:
This doesn't trigger a change in yarn.lock? I thought it did.

Reply:
Nope. Unlock `package-lock.json`, yarn doesnt track whether something is a dev or prod dependency. It references the `package.json` for that.. as it should, why keep 2 sources of truth.